misc/python/materialize/cli/ci_logged_errors_detect.py

```diff
@@ -127,7 +132,7 @@ def annotate_logged_errors(log_files: List[str]) -> None:
                         suite_name,
                         allow_multiple_subelements=True,
                     )
-                    test_case.add_failure_info(message=message, output=error.line)
+                    test_case.add_skipped_info(message=message, output=error.line)
```

Comment:
Can you explain this change, please?

Reply:
I'm trying to get an info box to appear in the annotations instead of an error for these. This didn't work, so I have to find another way, probably using `buildkite-agent annotate --style=info` directly.